examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -0,0 +1,854 @@
+      "source": [
+        "## Use the enrichment transform\n",
+        "\n",
+        "To use the [enrichment transform](https://beam.apache.org/releases/pydoc/2.54.0/apache_beam.transforms.enrichment.html#apache_beam.transforms.enrichment.Enrichment), [`EnrichmentHandler`](https://beam.apache.org/releases/pydoc/2.54.0/apache_beam.transforms.enrichment.html#apache_beam.transforms.enrichment.EnrichmentSourceHandler) is a required parameter. You can also use a configuration parameter to specify a `lambda` for a join function, a timeout, a throttler, and a repeater (retry strategy):\n",
+        "\n",
+        "\n",
+        "*   `join_fn`: A lambda function that takes dictionaries as input and returns an enriched row (`Callable[[Dict[str, Any], Dict[str, Any]], beam.Row]`). The enriched row specifies how to join the data fetched from the API. Defaults to a [cross-join](https://beam.apache.org/releases/pydoc/2.54.0/apache_beam.transforms.enrichment.html#apache_beam.transforms.enrichment.cross_join).\n",
+        "*   `timeout` - The number of seconds to wait for the request to be completed by the API before timing out. Defaults to 30 seconds.\n",
+        "*   `throttler` - Specifies the throttling mechanism. The only supported option is default client-side adaptive throttling.\n",
+        "*   `repeater` - Specifies the retry strategy when errors like `TooManyRequests` and `TimeoutException` occur. Defaults to [`ExponentialBackOffRepeater`](https://beam.apache.org/releases/pydoc/2.54.0/apache_beam.io.requestresponse.html#apache_beam.io.requestresponse.ExponentialBackOffRepeater).\n"
+      ],
+      "metadata": {
+        "id": "-Lvo8O2V-0Ey"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following example demonstrates the code needed to add this transform to your pipeline.\n",
+        "\n",
+        "\n",
+        "```\n",
+        "with beam.Pipeline() as p:\n",
+        "  output = (p\n",
+        "            ...\n",
+        "            | \"Enrich with BigTable\" >> Enrichment(bigtable_handler, timeout=10)\n",
+        "            | \"RunInference\" >> RunInference(model_handler)\n",
+        "            ...\n",
+        "            )\n",
+        "```\n",
+        "\n",
+        "\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "xJTCfSmiV1kv"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "To make a prediction, use the following features: `product_id`, `quantity`, `price`, `customer_id`, and `customer_location`. Retrieve the `customer_location` from the BigTable.\n",
+        "\n",
+        "Because the enrichment transform performs a [`cross_join`](https://beam.apache.org/releases/pydoc/2.54.0/apache_beam.transforms.enrichment.html#apache_beam.transforms.enrichment.cross_join) by default, design the custom join to enrich the input data. This design ensures that the join includes only the specified fields."
```

Comment:
```suggestion
        "Because the enrichment transform performs a [`cross_join`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment.html#apache_beam.transforms.enrichment.cross_join) by default, design the custom join to enrich the input data. This design ensures that the join includes only the specified fields."
```